### ISSUE TITLE
结果不对

### ISSUE THREAD

**Issue by Maoshib** [openmedlab/USFM#38]
为什么我直接拉取项目下来，按照你的readme里面的要求一步步操作，但是得到的结果始终达不到文献中的结果，我对模型没有以及配置没有进行任何修改

**Comment by XZheng0427** [comment]
According to the steps in the readme, the reproduced results are very different from the results in the paper and there is no test file?  Can the authors explain this?

**Comment by ruoyangliu** [comment]
请问有复现成功吗

**Comment by XZheng0427** [comment]
No, the results are pretty worse than the paper presented, even when using the pretrained weights.

**Comment by ruoyangliu** [comment]
that's tough. I am willing to use it as a feature extractor. Do you have any suggestions? Thanks

**Comment by hu2274898** [comment]
Has anyone successfully reproduced the results?

**Comment by sunyeongan** [comment]
@hu2274898 
I conducted tests on the BUSI data used as a follow-up to the paper (my data only used malignant and benign data, to test binary-class classification performance).



<img width="1588" height="163" alt="Image" src="https://github.com/user-attachments/assets/e1133d81-8ac7-416f-bc3f-3c7900e38b7a" />

<img width="2131" height="466" alt="Image" src="https://github.com/user-attachments/assets/265758a3-b656-4522-a604-19763f03fe94" />

**Comment by ShiDaizi** [comment]
> [@hu2274898](https://github.com/hu2274898) I conducted tests on the BUSI data used as a follow-up to the paper (my data only used malignant and benign data, to test binary-class classification performance).
> 
> <img alt="Image" width="1588" height="163" src="https://private-user-images.githubusercontent.com/44018024/514203137-e1133d81-8ac7-416f-bc3f-3c7900e38b7a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODA2ODMxMDYsIm5iZiI6MTc4MDY4MjgwNiwicGF0aCI6Ii80NDAxODAyNC81MTQyMDMxMzctZTExMzNkODEtOGFjNy00MTZmLWJjM2YtM2M3OTAwZTM4YjdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNjA1VDE4MDY0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxYTYyYzZjYzNjODY2MzVlNTQzZThkMTczODc1ZDA4MWQwNzMyYzAwODIyODc0ODY4NGUxNTYzYzEwZWZiNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.mpOJeRWJh_JG-I6mvIa-E40QDuan1IfbZwsgI-VzJfc"> <img alt="Image" width="2000" height="437" src="https://private-user-images.githubusercontent.com/44018024/514203795-265758a3-b656-4522-a604-19763f03fe94.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODA2ODMxMDYsIm5iZiI6MTc4MDY4MjgwNiwicGF0aCI6Ii80NDAxODAyNC81MTQyMDM3OTUtMjY1NzU4YTMtYjY1Ni00NTIyLWE2MDQtMTk3NjNmMDNmZTk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNjA1VDE4MDY0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkMWVlOTcxZWI1YTk5YTg2Y2Y2ZDYwMTQyNmI3YjA4NzZkZTlkYjMzMDhmMGE2ZjFjNjI0MzkxMDg3Y2M4ZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.47iNWIBcUwlmj6hXmiX5vDClFR2Us0dCAtnXjx-Mz2I">

@sunyeongan 
Hello, have you solved this performance issue?

I used an average-pooling linear probe and took 10% (randomly) of BUSI as the test set and 10% as validation set. I trained for 200 epochs with a learning rate of 1e-3 and weight decay of 1e-4, and obtained an accuracy of 0.7051.

<img width="810" height="285" alt="Image" src="https://github.com/user-attachments/assets/4b87b35f-1865-4e4d-a047-dd05833099d7" />
